layers/+tools/verb/README.org

```diff
@@ -0,0 +1,68 @@
+#+TITLE: Verb layer
+
```

Comment:
Add proper tags declaration, in your case it should be I assume #+TAGS: layer|tool